File: Python/URI/1038.py
vetor = [4.00, 4.50, 5.00, 2.00, 1.50]
linha = input().split(' ')

a = int(linha[0])
b = int(linha[1])

total = vetor[a-1]*b

print("Total: R$ {:.2f}".format(total))

File: php/URI/1038.php
<?php

$vetor = [4.00, 4.50, 5.00, 2.00, 1.50];
$linha = explode(' ', readline());

$total = $vetor[$linha[0]-1] * $linha[1];

echo "Total: R$ ". number_format($total, 2, ".", "", )."\n";

?>